lib/main.dart

```diff
@@ -0,0 +1,144 @@
+import 'package:flutter/material.dart';
+import 'new_todo.dart';
+import 'specific_todo.dart';
+import 'list_all.dart';
+
+void main(List<String> args) {
+  runApp(const App());
+}
+
+class App extends StatelessWidget {
+  const App({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Todo App',
+      theme: ThemeData(
+        appBarTheme: const AppBarTheme(
+          backgroundColor: Colors.black,
+          foregroundColor: Colors.white,
+        ),
+      ),
+      home: const Todo(),
+    );
+  }
+}
+
+class Todo extends StatefulWidget {
+  const Todo({super.key});
+
+  @override
+  State<Todo> createState() => _TodoState();
+}
+
+class _TodoState extends State<Todo> {
+  final PageController controller = PageController();
+
+  final List<String> _todoTexts = <String>[];
+  final List<bool> _markedDone = List<bool>.filled(
+      1000, false); // so this makes it work to 1000 after that it should break
```

Comment:
formatting suggestion:

```suggestion
  // so this makes it work to 1000 after that it should break
  final List<bool> _markedDone = List<bool>.filled(1000, false);
```

Reply:
Why should it break at 1000? How does it break? Does the app crash?